pkg/apiserver/registry/generic/storage.go

```diff
@@ -9,7 +9,9 @@ import (
 )
 
 func NewRegistryStore(scheme *runtime.Scheme, resourceInfo utils.ResourceInfo, optsGetter generic.RESTOptionsGetter) (*registry.Store, error) {
-	strategy := NewStrategy(scheme, resourceInfo.GroupVersion())
+	gv := resourceInfo.GroupVersion()
+	gv.Version = runtime.APIVersionInternal
+	strategy := NewStrategy(scheme, gv)
```

Comment:
it feels like we should fix the builder workflow vs making this change here

Reply:
I will totally follow your lead here :)  Just want to get the funky stuff like this out of the feature branch if it should be normal :)

Reply:
yeah, makes sense. i'll follow up in slack